template_config.yml

```diff
@@ -27,6 +27,7 @@ check_stray_pulpcore_imports: true
 cherry_pick_automation: false
 ci_env:
   GITHUB_USER: ${{ github.event.pull_request.user.login }}
+  LOCK_REQUIREMENTS: 0
```

Comment:
Just an FYI, this will get reverted before the `feature/rbac-roles` branch gets merged into master.